modules/migration-configuring-mcg.adoc

```diff
@@ -46,5 +44,8 @@ aws_secret_access_key=<AWS_SECRET_ACCESS_KEY>
 EOF
 ----
 +
-You use the `credentials-velero` file to create a `Secret` object when you install the Data Protection Application.
-endif::[]
+You use the `credentials-velero` file to create a `Secret` object when you install the Data Protection Application. 
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
@subhtk This can't go in the module, since we aren't allowed xrefs in modules.

You'll need to move this to the assembly that includes the module. It looks like there are a few assemblies that include this module, so you'll want to add them to all of them.

For example, in `migration_toolkit_for_containers/installing-mtc.adoc`, you would add this around line 80, right after the `include::modules/migration-configuring-mcg.adoc[leveloffset=+2]` line.

And you'll also need to reformat the xref to be an actual cross-reference, and not a full file link. I'm actually surprised it doesn't fail the build. But right now how you have it, it opens the file on the portal. It should open the file relative to where you are, so when we're testing on docs.openshift.com, it should open the file on docs.openshift.com.

So the path should be a relative path. I didn't test it, but it should look something like:

```
xref:../backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.adoc#oadp-about-backup-snapshot-locations_installing-oadp-aws[About backup and snapshot locations and their secrets]
```

Reply:
Understood, I will update the doc accordingly. And I will check with the links as well.